### DISCUSSION TITLE
Can I template `jj log` to show me the first three files changed?

### DISCUSSION THREAD

**Discussion by ocharles** [jj-vcs/jj#5092] [Q&A] [answered]
I quite often end up with changes with no description, but I know these changes are important - and one of them is probably me fiddling with a particular file. Is there a way I can template `jj log` to show (at most) three changes? I don't care which. So far I've got `jj log -T 'self.diff().summary()'` which is a start! I just don't know how to take the first 3 lines

**Comment by yuja** [comment, marked as answer]
No, there isn't line truncation function nor function that returns a list of changed files yet. `truncate_end()` is closer, but it strips contents based on width.

**Reply by ocharles** [reply]
Shoot. Thanks! I'll open a feature request (if there isn't already one).
